docs/resources/dws_cluster.md

```diff
@@ -118,6 +118,8 @@ The following arguments are supported:
 
 * `elb_id` - (Optional, String) Specifies the ID of the ELB load balancer.
 
+* `lts_flag` - (Optional, Bool) Specified whether to enable lts-logs.
```

Comment:
The description in the document is inconsistent with the description of the schema definition.

Reply:
done